calculator.js

```diff
@@ -0,0 +1,33 @@
+var calculator = {
+
+
+
+
+  addition: function add(num1, num2){
+    return num1 + num2;
+    // console.log(addtion(num1, num2));
+  },
+subt: function sub(num1, num2) {
```

Comment:
Also make sure your indentation is correct and consistent. We always indent the items in an object at the same level.